<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color: rgb(0, 0, 0);
}

QLabel{
color:rgb(168,168,168)
}

QLineEdit{
border: 1px solid rgb(100,100,100);
background-color:rgb(230,230,230);
}

QTabWidget::pane { border: 0; }
QTabBar::tab{background: black;
color: rgb(168,168,168);
}
QTabBar::tab:selected {background: rgb(50,50,50);}

QPushButton{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:disabled{
color: rgb(80,80,80);
background-color:rgb(0,0,0);
border: 1px solid rgb(80,80,80);
border-radius: 5px
}
</string>
   </property>
   <widget class="QFrame" name="settings_frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>250</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="push_Servers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
border: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>86</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature Log:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ImageNum">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>140</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="push_all_off">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>103</y>
       <width>180</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ALL OFF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Plotted time (h): </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_time">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_delay">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample delay (s):</string>
     </property>
    </widget>
    <widget class="QFrame" name="loop_1_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>250</width>
       <height>175</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#loop_1_frame{
background-color: rgb(0, 0, 0);
border: 2px solid white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="loop_2_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>325</y>
       <width>250</width>
       <height>175</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#loop_2_frame{
background-color: rgb(0, 0, 0);
border: 2px solid white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="temperature_frame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>350</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
